README.md

```diff
@@ -48,7 +48,7 @@ Reference the Bootstrap CSS stylesheet in `public/index.html` (optional):
 
 In the app directory, generate the files for the resource you want:
 
-    $ api-platform-generate-crud http://localhost src/ --resource foo
+    $ api-platform-generate-crud https://demo.api-platform.com/index.jsonld src/ --resource foo
```

Comment:
https://demo.api-platform.com/ is enough for the application but as a user trying to understand what is happening behind the scenes it is quite misleading. Clicking on the link in chrome takes you to the swagger documentation. When I did this I erroneously assumed that the Entrypoint was the same as the documentation link and spent time trying to point the generator at my own implementation which has a modified path the the documentation. 

If the example documentation had shown https://demo.api-platform.com/index.jsonld I would visited that link and seen the Entrypoint of the demo system, and with that enlightenment would have been able to save a lot of time.